app/services/analytics_events.rb

```diff
@@ -807,6 +807,36 @@ def idv_phone_otp_delivery_selection_submitted(
     )
   end
 
+  # User visited idv phone of record
+  def idv_phone_of_record_visited
+    track_event('IdV: phone of record visited')
+  end
+
+  # User visited idv phone OTP delivery selection
+  def idv_phone_otp_delivery_selection_visit
+    track_event('IdV: Phone OTP delivery Selection Visited')
+  end
+
+  # @param [String] step the step the user was on when they clicked use a different phone number
+  # User decided to use a different phone number in idv
+  def idv_phone_use_different(step:, **extra)
+    track_event(
+      'IdV: use different phone number',
+      step: step,
+      **extra,
+    )
+  end
+
+  # User completed idv
+  def idv_review_complete
+    track_event('IdV: review complete')
+  end
+
+  # User visited idv phone of record
```

Comment:
The review step is where the user reviews the PII we have for them right?

Reply:
No this is actually after they encrypt their password.  It's very misleading.  'IdV: final resolution' was created to make it clear.

Reply:
Ok but it looks like this description was copy-pasted... both `idv_phone_of_record_visited` and `idv_review_info_visited` have the comment "User visited idv phone of record"